db/queries.js

```diff
@@ -11,7 +11,29 @@ const addBooking = (date, type, id, lat, lon) => {
 	return pool.query(queryStr, [date, type, id, lat, lon]).then((response) => response);
 };
 
+const fetchFollows = (user_id) => {
+	const queryStr = `SELECT json_agg(artist_id) FROM follows WHERE fan_id=$1`;
+	return pool.query(queryStr, [user_id]).then((response) => response);
+};
+
+const fetchTourdates = (artist_id, start_date, end_date) => {
+	const queryStr = `SELECT user_name AS artist_name, bio, portrait_url, website,
+	  (
```

Comment:
The spacing looks weird in this querystr, mainly the parenthesis aren't lining up

Reply:
For some reason VS code was adding tabs at the beginning of lines instead of two spaces. I changed it locally and will include the change in my next commit.